### ISSUE TITLE
'wo RL' path problem

### ISSUE THREAD

**Issue by XinyiChu** [uzh-rpg/rl_vo#6]
May I ask, during the test phase, if I want to test the results without RL as done in the paper, where does the 'wo RL' path in evaluate_runs.py come from?

**Comment by ahengn** [comment]
> May I ask, during the test phase, if I want to test the results without RL as done in the paper, where does the 'wo RL' path in evaluate_runs.py come from?

change the use_rl_actions to False in the config_eval.yaml and run the play.py

**Comment by GarlanLou** [comment]
Have you found that  pinhole configuration file?

**Comment by XinyiChu** [comment]
> > May I ask, during the test phase, if I want to test the results without RL as done in the paper, where does the 'wo RL' path in evaluate_runs.py come from?
> 
> change the use_rl_actions to False in the config_eval.yaml and run the play.py

>thank u!

**Comment by XinyiChu** [comment]
> Have you found that pinhole configuration file?

>sorry, i can't find the pinhole configuration file, i use the svo_test_pinhole.yaml

**Comment by GarlanLou** [comment]
> > Have you found that pinhole configuration file?
> 
> > sorry, i can't find the pinhole configuration file, i use the svo_test_pinhole.yaml

Can you provide a contact information for me to communicate together?

**Comment by XinyiChu** [comment]
> > > Have you found that pinhole configuration file?
> > 
> > 
> > > sorry, i can't find the pinhole configuration file, i use the svo_test_pinhole.yaml
> 
> Can you provide a contact information for me to communicate together?

>do u mean wechat or e_mail?

**Comment by GarlanLou** [comment]
> > > > Have you found that pinhole configuration file?
> > > 
> > > 
> > > > sorry, i can't find the pinhole configuration file, i use the svo_test_pinhole.yaml
> > 
> > 
> > Can you provide a contact information for me to communicate together?
> 
> > do u mean wechat or e_mail?

yes, you can send your wechat id to my mailbox which can be found on my profile.

**Comment by XinyiChu** [comment]
> > > > > Have you found that pinhole configuration file?
> > > > 
> > > > 
> > > > > sorry, i can't find the pinhole configuration file, i use the svo_test_pinhole.yaml
> > > 
> > > 
> > > Can you provide a contact information for me to communicate together?
> > 
> > 
> > > do u mean wechat or e_mail?
> 
> yes, you can send your wechat id to my mailbox which can be found on my profile.

> I have send my wechat id to your mailbox

**Comment by AthorGates** [comment]
> Can you provide a contact information for me to communicate together?

**Comment by AthorGates** [comment]
> > May I ask, during the test phase, if I want to test the results without RL as done in the paper, where does the 'wo RL' path in evaluate_runs.py come from?
> 
> change the use_rl_actions to False in the config_eval.yaml and run the play.py

>Can you provide a contact information for me to communicate together?

**Comment by XinyiChu** [comment]
> > May I ask, during the test phase, if I want to test the results without RL as done in the paper, where does the 'wo RL' path in evaluate_runs.py come from?
> 
> change the use_rl_actions to False in the config_eval.yaml and run the play.py
>Can you provide a contact information for me to communicate together?

**Comment by glt621** [comment]
> > > 请问，在测试阶段，如果我想像论文中那样测试不使用RL的结果，那么evaluate_runs.py中的‘wo RL’路径从哪里来？
> > 
> > 
> > 在 config_eval.yaml 中将 use_rl_actions 改为 False 并运行 play.py
> > 你能提供一个联系方式让我可以一起交流吗？

可以发送到我的邮箱吗
